app/index.js

```diff
@@ -1,24 +1,32 @@
 var OSinfo = require('../modules/OSinfo');
 
 process.stdin.setEncoding('utf-8');
-process.stdin.on('readable', function(){
+process.stdin.on('readable', function () {
     var input = process.stdin.read();
-    if(input !== null){
+    if (input !== null) {
         var instruction = input.trim();
-        switch(instruction){
-            case '/exit':
-                process.stdout.write('Quitting app!\n');
-                process.exit();
-                break;
-            case '/sayhello':
-                process.stdout.write('hello!\n');
-                break;
-            case '/getOSinfo':
-                OSinfo.print();
-                break;
-            default:
-                process.stderr.write('Wrong instruction!\n');
-        }
     }
-});
+    switch (instruction) {
```

Comment:
your switch is still inside if ;) nesting is not indantation :) try to write it by changing if statement